ssg/oval_object_model/oval_entities/object.py

```diff
@@ -0,0 +1,47 @@
+from ..general import (
+    OVAL_NAMESPACES,
+    STR_TO_BOOL,
+    OVALEntity,
+    load_OVAL_entity_property,
+    load_notes,
+    required_attribute,
+)
+
+
+def load_object(oval_object_xml_el):
+    notes_el = oval_object_xml_el.find("./{%s}notes" % OVAL_NAMESPACES.get("oval_def"))
+
+    object_property = []
+    for child_node_el in oval_object_xml_el:
+        if notes_el == child_node_el:
+            continue
```

Comment:
What is the reason for this special handling of notes element?

Reply:
Because the notes element is represented as a class. The notes element is loaded using the `load_notes()` function.

Reply:
I think it would need at least a comment there.